packages/fxa-content-server/app/scripts/templates/partial/account-suggestion.mustache

```diff
@@ -1,6 +1,10 @@
 {{#showAccountSuggestion}}
-  <div class="info mx-auto mb-5" id="suggest-account">
-      {{#unsafeTranslate}}Why do I need to create this account? <a %(escapedSuggestionAttrs)s>Find out here</a>{{/unsafeTranslate}}
-      <span class="dismiss" tabindex="-1">&#10005;</span>
+  <div class="info" id="suggest-account">
+    <p class="info-text">
+      {{#unsafeTranslate}}Why do I need to create this account? <a %(escapedSyncSuggestionAttrs)s>Find out here</a>{{/unsafeTranslate}}
+    </p>
+    <button type="button" class="dismiss" aria-label="Close">
+      <img src="/images/close.svg" alt="&#10005;" aria-hidden="true"/>
+    </button>
```

Comment:
@pdehaan I've been thinking about how to improve this component from an a11y perspective, and have taken a stab at a different structure. With this new approach, screen readers would read out "close, button" and the image alt would be ignored.